build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -207,6 +207,20 @@ service ActionCache {
 // `{instance}/uploads/{uuid}/blobs/{hash}/{size}/foo/bar/baz.cc`. Anything
 // after the `size` is ignored.
 //
+// Clients can upload compressed data with a `resource_name` of the form
```

Comment:
Can you restructure this a little bit to make it 100% clear that the same comments on uuid and trailing filename also apply to compressed-blobs?

Reply:
Done. While doing so, I dropped the part which described a potential use for the optional metadata.